scripts/sanity_chk/sanitylib.py

```diff
@@ -3350,7 +3350,7 @@ def add_ignore_file(self, pattern):
         self.ignores.append('*' + pattern + '*')
 
     def add_ignore_directory(self, pattern):
-        self.ignores.append(pattern + '/*')
+        self.ignores.append('*/' + pattern + '/*')
```

Comment:
In the option below you have `(".*/" + pattern + '/.*')`. Here it is `('*/' + pattern + '/*')` so it looks like `.` might be missing before `*`. Can this be the reason why @tribal-tec is observing problems with `lcov`?

Reply:
`lcov` and `gcov` have both different syntax for specifying ignore patterns. One requires `*/` and the other requires `.*/` to get the same effect.